CondTools/Ecal/src/EcalDCSHandler.cc

```diff
@@ -272,6 +272,8 @@ bool popcon::EcalDCSHandler::insertHVDataSetToOffline(const std::map<EcalLogicID
         ex_y[4] = 12;
         ex_x[5] = 6;
         ex_y[5] = 11;
+      } else {
+        std::cout << "Dee number not known !" << std::endl;
```

Comment:
This will not fix the issue about `ex_x[l]` containing garbage since the array is still not defined. And indeed the same is true for `ex_y[l]` as far as I can see even though the SA does not flag it.

Since we have only four Dees in CMS I think you could just throw an exception since clearly something is wrong in this case.